src/Symfony/Component/VarDumper/Caster/Caster.php

```diff
@@ -19,6 +19,8 @@
  * @author Nicolas Grekas <p@tchwork.com>
  *
  * @final
+ *
+ * @internal since Symfony 7.3
```

Comment:
I wouldn't mark this one as internal

Reply:
This base class is indeed not internal. Its constants are part of the public API. However, some of its method might be tagged as internal

Reply:
castPhpIncompleteClass yes

Reply:
Isn't `castObject` also internal ? Custom casters receive the output of this method as their second argument, so there should be no reason to call it directly outside the AbstractCloner

Reply:
ah yes right